client/dashboard/data/index.ts

```diff
@@ -361,6 +362,48 @@ export const restoreSitePlanSoftware = async ( siteIdOrSlug: string ) => {
 	} );
 };
 
+export const siteOwnerTransfer = async (
+	siteIdOrSlug: string,
+	data: { new_site_owner: string }
+) => {
+	return wpcom.req.post(
+		{
+			path: `/sites/${ siteIdOrSlug }/site-owner-transfer?calypso_origin=${ window.location.origin }`,
```

Comment:
Any good reason to pass the query arg here rather than in the second `wpcom.req.post()` argument?

Reply:
Oh, no, I meant `wpcom.req.post()` supports a different signature: `wpcom.req.post( path, query, body )`. 

cc @jsnajdr since I know he was doing some consistency work there to make the use simpler.

Reply:
It works as expected 🙂